streams/src/test/java/org/apache/kafka/streams/integration/GlobalKTableIntegrationTest.java

```diff
@@ -261,16 +265,23 @@ public boolean conditionMet() {
         }, 30000L, "waiting for initial values");
         System.out.println("no failed test");
     }
+    
+    @Test(timeout = 10000)
+    public void shouldRestoreAbortedMessages() throws ExecutionException, InterruptedException {
```

Comment:
nit: the name is a little miss leading -- aborted message should not be restored...

It also seems, this test only checks if the restore terminates -- I am wondering, if we should also write committed data into the topic and check that the messages got restored? Not sure if necessary.